support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -58,4 +60,24 @@ export const tests: Tests = {
     targetPage: allLandingPagesAndThankyouPages,
     seed: 18,
   },
+  priceCardsInHeroTest: {
+    variants: [
+      {
+        id: 'control',
+      },
+      {
+        id: 'variant',
+      },
+    ],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    isActive: true,
+    referrerControlled: false,
+    targetPage: digiSubLandingPagesNotAus,
```

Comment:
we'll need to add an `optimizeId` value in here once we have the test set up in Optimize